matrix/createMatrix.py

```diff
@@ -0,0 +1,31 @@
+from number.NumberFabric import NumberFabric
+from number.NumberType import NumberType
+from matrix.MyMatrix import MyMatrix
+import numpy
+import random
+
+size = 10 # z czego size potem będzie w pętli, od 0 do jakiejś tam dużej liczby (WAŻNE PRZY TESTACH)
```

Comment:
To może dać go jako argument?

Reply:
Oh fuck 
On już jest podawany jako argument, po prostu zapomniałam wywalić tej linii jak to przenosiłam z maina